src/service.js

```diff
@@ -10,10 +10,20 @@
     app.service('vcRecaptchaService', ['$window', '$q', function ($window, $q) {
         var deferred = $q.defer(), promise = deferred.promise, recaptcha;
 
-        $window.vcRecaptchaApiLoaded = function () {
+        if(angular.isUndefined($window.vcRecaptchaApiLoaded)){
```

Comment:
This if block can become `$window.vcRecaptchaApiLoaded = $window.vcRecaptchaApiLoaded || [];`